### PR TITLE
Initial local testing

### PR DESCRIPTION
**Pull request by ritchelong**
I manually load with a path in init. Feel free to change that to load files. Currently having issues with curling

### CODE REVIEW

File: flaskphiid/__init__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 # load medlp interface
 from compmed.CompMedServiceInterface import CompMedServiceInterface
-import utils.json_parser_utils as JSONParser
+from utils import json_parser_util as JSONParser
```
Comment (esilgard) [inline flaskphiid/__init__.py:19]: does this still work?
it looks like you brought the utils folder into the flaskphiid to test it out and I'm assuming this import worked then - but does it when you take it out?

Comment (ritchelong) [reply flaskphiid/__init__.py:19]: i brought the utils folder out but ended up not using it. instead i used a renamed utils installed on the compmed setup as compmed_utils to import json_parser. 

it should still work, i just was testing out if there was a conflict with the utils name on my end

Comment (esilgard) [reply flaskphiid/__init__.py:19]: gotcha - you updated the ComprehendMedicalInterface codebase as well?

**Review by esilgard — APPROVED** [review]
lets do it -

File: flaskphiid/run.py [hunk 2]
```diff
@@ -14,4 +14,4 @@
                             help="Set port. Default is 5000 ")
 cmd_args = cmd_arg_parser.parse_args()
 
-app.run(ssl_context='adhoc', host=cmd_args.endpoint, port=cmd_args.port)
```
Comment (esilgard) [inline flaskphiid/run.py:17]: let's leave this in but commented out

**Pull request merged**